r-package/build_package.R

```diff
@@ -6,15 +6,31 @@
 # To prepare a CRAN build:
 #   `Rscript build_package.R --as-cran`
 
+Sys.setenv(LINTR_COMMENT_BOT="false")
```

Comment:
I forgot to mention this last time -- I think we should move this to the travis script  through something like `export LINTR_COMMENT_BOT=false`, because it's not relevant when building locally, and it's nice to avoid touching local environment variables when possible.